spec/fixtures/sshd_config_debian

```diff
@@ -97,9 +97,20 @@ GSSAPICleanupCredentials yes
 UsePAM yes
 
 # Accept locale-related environment variables
-AcceptEnv LANG LC_CTYPE LC_NUMERIC LC_TIME LC_COLLATE LC_MONETARY LC_MESSAGES
```

Comment:
You can't change the fixtures files, they are meant to be static

Reply:
I can't see any problem on it. It is according the module will generate the files.